packages/client/components/ActivityLibrary/ActivityDetails/ActivityDetails.tsx

```diff
@@ -22,33 +20,26 @@ graphql`
     category
     orgId
     teamId
+    illustrationUrl
     isFree
     scope
-    team {
-      editingScaleId
-      ...PokerTemplateScaleDetails_team
```

Comment:
I think at some point this got refactored to TemplateDetails? Either way, removed it since it wasn't used in this specific component